src/App.js

```diff
@@ -1,25 +1,31 @@
-import logo from './logo.svg';
-import './App.css';
+import React, {Component} from 'react';
+import Header from './components/Header';
+import Feed from './components/Feed';
+import SignIn from './components/SignIn';
+import SignUp from './components/SignUp';
+import Profile from './components/Profile';
+import Search from './components/Search';
+import {BrowserRouter, BrowserRouter as Router, Route, Routes} from 'react-router-dom';
+import "./bootstrap.css";
+import { ToastContainer } from "react-toastify";
+import axios from "axios";
+axios.defaults.baseURL = "http://127.0.0.1:8000";
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+    return (
+        <BrowserRouter>
+            <div className="App">
+                <Header/>
+                    <Routes>
+                        <Route path="/feed" element={<Feed />} />
+                        <Route path="/profile" element={<Profile />} />
+                        <Route path="/signin" element={<SignIn />} />
+                        <Route path="/signup" element={<SignUp />} />
+                        <Route path="/search" element={<Search />} />
+                    </Routes>
+            </div>
+        </BrowserRouter>
+    );
 }
 
-export default App;
+export default App;
```

Comment:
Please add a newline here and in some other files

Reply:
Done